src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -295,6 +307,11 @@ protected override void Dispose(DisposeTypes type)
                 //Release your own managed resources here.
                 //You should release all of your own disposable objects here.
                 RemoveVisual("loadingImageVisual");
+
+                if (defaultLottieView != null)
+                {
+                    Utility.Dispose(defaultLottieView);
+                }
```

Comment:
seems
```
defaultLottieView = null;
```
is missing?

Reply:
I agree with @everLEEst .
It seems that defaultLottieView = null; is necessary.

Reply:
Okay, thank you for your reviews. :+1: